README.md

```diff
@@ -129,6 +129,46 @@ func testActorQueueOrdering() async {
 }
 ```
 
+### Sending ordered asynchronous tasks to the `@MainActor` from a nonisolated context
+
+Use a `MainActorQueue` to send ordered asynchronous tasks to the `@MainActor`’s isolated context from nonisolated or synchronous contexts. Tasks sent to this queue type are guaranteed to begin executing in the order in which they are enqueued. Like an `ActorQueue`, execution order is guaranteed only until the first [suspension point](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html#ID639) within the enqueued task. A `MainActorQueue` executes tasks within the its adopted actor’s isolated context, resulting in `MainActorQueue` task execution having the same properties as a `@MainActor`'s' code execution: code between suspension points is executed atomically, and tasks sent to a single `MainActorQueue` can await results from the queue without deadlocking.
```

Comment:
```suggestion
Use a `MainActorQueue` to send ordered asynchronous tasks to the `@MainActor`’s isolated context from nonisolated or synchronous contexts. Tasks sent to this queue type are guaranteed to begin executing in the order in which they are enqueued. Like an `ActorQueue`, execution order is guaranteed only until the first [suspension point](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html#ID639) within the enqueued task. A `MainActorQueue` executes tasks within its adopted actor’s isolated context, resulting in `MainActorQueue` task execution having the same properties as a `@MainActor`'s' code execution: code between suspension points is executed atomically, and tasks sent to a single `MainActorQueue` can await results from the queue without deadlocking.
```
typo.

Reply:
Also not sure if the second half of that explanation is needed, may work out better to just lead with the behavior rather than re-explaining `ActorQueue`

Reply:
Good catch on the typo! And fair that the documentation is repetitive here but sometimes repetition is helpful: I'm trying to drive home that *this type behaves the same way an `ActorQueue` does*, and also trying to help folk who might only be using a `MainActorQueue` and not utilizing the other types – you shouldn't have to read the whole README to have an understanding of what's happening in this type imo.

Reply:
I get what you mean. As a possible alternative, is there some way to link to the original in the same README?

Not sure how this particular breed of markdown works here 😬

Reply:
Yes, but only the header which is a bit far from the thing I want to describe